SPECS/gsm/gsm-makefile.patch

```diff
@@ -15,7 +16,7 @@ diff -up gsm-1.0-pl17/Makefile.mk gsm-1.0-pl17/Makefile
  
 +SO_MAJOR = 1
 +SO_MINOR = 0
-+SO_PATCH = 18
++SO_PATCH = 19
```

Comment:
Should this be 22 as well? Did we hand-port this patch or did we get an updated copy of the patch from somewhere?

Reply:
Ofcourse .. thank you :)